WordPress/Classes/Extensions/URL+Helpers.swift

```diff
@@ -2,6 +2,18 @@ import Foundation
 
 extension URL {
 
+    /// The URLResource fileSize of the file at the URLs, if available.
+    ///
+    /// - param URL: A file URL.
+    ///
+    var resourceFileSize: Int? {
```

Comment:
>fileSize maybe will be a better name.

I'd prefer to keep the `resource` keyword as it includes a bit more information as to where this value is/should be coming from.

>And is Int large enough to hold the result?

I did question that, but the `fileSize` attribute does indeed return an `Int`: https://developer.apple.com/reference/foundation/urlresourcevalues/1779734-filesize

>The parameter documentation also needs to be updated. What is the result of it? bytes? Kbs? and we no longer need the param info.

Documentation updated.